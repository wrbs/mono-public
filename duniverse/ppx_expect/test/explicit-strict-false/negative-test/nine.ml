(* Show that, even when compiling with [-expect-test-strict-indentation=false], the issued
   corrected files makes the "fixed" expect blocks satisfy standard indentation rules *)

let%expect_test _ =
  let module _ = struct
    let () =
      print_string "hello";
      [%expect
        {|
                                              goodbye
                                            |}]
    ;;

    let () =
      print_string "hello\n";
      [%expect
        {|
                                              goodbye
                                            |}]
    ;;

    let () =
      print_string "hello\n\n";
      [%expect
        {|
                                              goodbye

                                            |}]
    ;;

    let () =
      print_string "\nhello";
      [%expect {|

                                              goodbye|}]
    ;;

    let () =
      print_string "\nhello\n";
      [%expect
        {|

                                              goodbye
                                            |}]
    ;;

    let () =
      print_string "\nhello\n\n";
      [%expect
        {|

                                              goodbye

                                            |}]
    ;;

    let () =
      print_string "\n\nhello";
      [%expect {|


                                              goodbye|}]
    ;;

    let () =
      print_string "\n\nhello\n";
      [%expect
        {|


                                              goodbye
                                            |}]
    ;;

    let () =
      print_string "\n\nhello\n\n";
      [%expect
        {|


                                              goodbye

                                            |}]
    ;;
  end
  in
  ()
;;
