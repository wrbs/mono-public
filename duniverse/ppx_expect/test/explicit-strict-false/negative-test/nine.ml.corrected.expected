(* Show that, even when compiling with [-expect-test-strict-indentation=false], the issued
   corrected files makes the "fixed" expect blocks satisfy standard indentation rules *)

let%expect_test _ =
  let module _ = struct
    let () =
      print_string "hello";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "hello\n";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "hello\n\n";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "\nhello";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "\nhello\n";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "\nhello\n\n";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "\n\nhello";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "\n\nhello\n";
      [%expect {| hello |}]
    ;;

    let () =
      print_string "\n\nhello\n\n";
      [%expect {| hello |}]
    ;;
  end
  in
  ()
;;
